BUILDING.md

```diff
@@ -1,6 +1,6 @@
 # Building GAPID
 
-GAPID uses the [Bazel build system](https://bazel.build/). The current minimum required version of bazel is 0.18.1.
+GAPID uses the [Bazel build system](https://bazel.build/). The current minimum required version of bazel is 0.25.1.
```

Comment:
Gross big jump. But I assume its basically impossible to avoid.

Reply:
The reality is that we were building with 0.20 before, but yeah, we should keep up with it a bit more.